client/src/components/AuditAdmin/Progress/Progress.tsx

```diff
@@ -195,30 +195,48 @@ const Progress: React.FC<IProgressProps> = ({
             return null
         }
       },
-      sortType: sortByRank(
-        ({ currentRoundStatus, ballotManifest, batchTallies, cvrs }) => {
-          if (!currentRoundStatus) {
-            const files: IFileInfo['processing'][] = [ballotManifest.processing]
-            if (batchTallies) files.push(batchTallies.processing)
-            if (cvrs) files.push(cvrs.processing)
-
-            const numComplete = files.filter(
-              f => f && f.status === FileProcessingStatus.PROCESSED
-            ).length
-            const anyFailed = files.some(
-              f => f && f.status === FileProcessingStatus.ERRORED
-            )
-            if (anyFailed) return 0
-            if (numComplete === 0) return -1
-            return numComplete
-          }
-          return {
-            [JurisdictionRoundStatus.NOT_STARTED]: 0,
-            [JurisdictionRoundStatus.IN_PROGRESS]: 1,
-            [JurisdictionRoundStatus.COMPLETE]: 2,
-          }[currentRoundStatus.status]
+      sortType: sortByRank((jurisdiction: IJurisdiction) => {
+        const {
+          currentRoundStatus,
+          ballotManifest,
+          batchTallies,
+          cvrs,
+        } = jurisdiction
+        const progressStatus = getJurisdictionStatus(
+          jurisdiction,
+          lastLoginQuery.data![jurisdiction.id]
+        )
+        const hasLoggedIn = ![
+          JurisdictionProgressStatus.UPLOADS_NOT_STARTED_NO_LOGIN,
+          JurisdictionProgressStatus.AUDIT_NOT_STARTED_NO_LOGIN,
+        ].includes(progressStatus)
+
+        if (!currentRoundStatus) {
+          const files: IFileInfo['processing'][] = [ballotManifest.processing]
+          if (batchTallies) files.push(batchTallies.processing)
+          if (cvrs) files.push(cvrs.processing)
+
+          const numComplete = files.filter(
+            f => f && f.status === FileProcessingStatus.PROCESSED
+          ).length
+          const anyFailed = files.some(
+            f => f && f.status === FileProcessingStatus.ERRORED
+          )
+          if (anyFailed) return hasLoggedIn ? 0 : -1
+          if (numComplete === 0) return hasLoggedIn ? -1 : -2
+          return numComplete
         }
-      ),
+        return {
+          [JurisdictionRoundStatus.NOT_STARTED]: () => {
+            if (hasLoggedIn) {
+              return 0
```

Comment:
This preserves the behavior that the following cases are weighted equally when a login exists:

1. Round not started, uploads attempted but failed
2. Round started but no action taken in audit

IMO case 2 should have higher sort weight than case 1, but I don't have context on the previous decision to weight them equally.

Reply:
Good q! I believe the context here is that round starts are audit-wide and apply across jurisdictions. It isn't possible for a round to be started for just a subset of jurisdictions

I might be misunderstanding what you're pointing out though

A comment describing the complete hierarchy here could be helpful! I'm having a bit of a hard time parsing (already was with the prior code and we've understandably had to add even more logic)

Reply:
That answers my question, thanks. Comment and tests for this behavior added in https://github.com/votingworks/arlo/pull/2061/commits/2e277b6fc66f4f04ebbc77e33738718ae43e9caa